app/scripts/lib/ppom/ppom-middleware.ts

```diff
@@ -69,23 +69,27 @@ export function createPPOMMiddleware(
         // eslint-disable-next-line require-atomic-updates
         const securityAlertId = uuid();
 
-        ppomController.usePPOM(async (ppom: PPOM) => {
```

Comment:
Why was this change needed?

Reply:
Even though `ppom.validateJsonRpc(req)` returns  `securityAlertResponse` we need to return it to PPOM controller, because inside PPOM controller, we need to add the infura rpc calls and the number of times they were called for the request (`providerRequestsCount`) to `securityAlertResponse` and then return the updated object. Otherwise `securityAlertResponse` won't have the `providerRequestsCount` objects.

Reply:
:+1: